packages/cli/README.md

```diff
@@ -369,8 +371,9 @@ Fields:
 |splashScreenFadeOutDuration|number|true|Duration for the splash screen fade out animation.|
 |startUrl|string|true|The start path for the TWA. Must be relative to the domain.|
 |themeColor|string|true|The color used for the status bar.|
-|webManifestUrl|string|false|Full URL to the PWA Web Manifest. Required for the application to be compatible with Chrome OS devices.|
- 
+|webManifestUrl|string|false|Full URL to the PWA Web Manifest. Required for the application to be compatible with Chrome OS and Meta Quest devices.|
+|fullScopeUrl|string|false|The navigation scope that the browser considers to be within the app. If the user navigates outside the scope, it reverts to a normal web page inside a browser tab or window. Must be a full URL. Required only for Meta Quest devices.|
```

Comment:
`fullScopeUrl` is not a flag, it's a regular field (like `startUrl` or `webManifestUrl`). This field is only read by the Meta Quest Browser on Meta Quest devices and just ignored by other devices as unknown.

Reply:
I'm wary that if the user uses `fullScopeUrl` but not `isMetaQuest` they're doing something wrong. It seems more likely they they meant to add Quest support, but missed the `isMetaQuest` flag, than they intentionally added `fullScopeUrl` for no reason.

Reply:
@PEConn What exactly do you suggest?

We cannot use the presence of `fullScopeUrl` to throw an error.

The problem is that currently `fullScopeUrl` will be added to `twa-manifest.json` in any case, regardless of the `isMetaQuest` flag value. But if `isMetaQuest` is `false` or missing, the `fullScopeUrl` will not be used in any way.

That's why I've added `Required only for Meta Quest devices.` to the field description.

BTW, I didn't add the `minSdkVersion` here. My rationale: because it shouldn't be changed by hand. Should I add it?

This is the problem of related (computed) fields.

If `isChromeOSOnly` is `true`, then `webManifestUrl` must be present. But I can edit `twa-manifest.json` in text editor and remove `webManifestUrl` or just use the old `twa-manifest.json` (generated by the old version of Bubblewrap) that has no `webManifestUrl`. In this case, we don't throw any build errors.

E.g. what stops the user from setting `isMetaQuest` to `true` and e.g. changing `minSdkVersion` back from `23` to `19` or something else?

As an idea, I can suggest to improve the description. What about adding something like `To use this field, the 'isMetaQuest' flag must be 'true'` to the `fullScopeUrl` description? Will it solve the problem?

Reply:
Ah OK, I misunderstood where `fullScopeUrl` was coming from - yeah, if it's going to be in `twa-manifest.json` regardless there's not anything we can do. Things are fine as is.